dev/oracle/latest/stop.sh

```
@@ -0,0 +1,10 @@
+# Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved
+
+#!/usr/bin/env bash
+set -Eeuxo pipefail # https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
+cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" # https://stackoverflow.com/a/17744637
+
+
+docker compose -p sequelize-oracle-latest down --remove-orphans
+
+echo "Local latest supported Oracle DB instance stopped (if it was running).
```

Shellcheck (0.11.0)

[error] Couldn't parse this double quoted string. Fix to allow more checks.

(SC1073)